cmake/BLASFinder.cmake

```diff
@@ -272,6 +280,20 @@ if (test_all OR test_default)
     debug_print_list( "default" )
 endif()
 
+#---------------------------------------- Intel OneMKL
+if (test_all OR test_onemkl)
+    # IntelLLVM compiler + OpenMP: require intel_thread library.
+    if (test_int)
+        list( APPEND blas_name_list "Intel OneMKL lp64,  Intel threads (iomp5), ifort")
+        list( APPEND blas_libs_list "-lmkl_intel_lp64  -lmkl_intel_thread -lmkl_core" )
+    elseif (test_int64)
+        list( APPEND blas_name_list "Intel OneMKL ilp64, Intel threads (iomp5), ifort")
+        list( APPEND blas_libs_list "-lmkl_intel_ilp64 -lmkl_intel_thread -lmkl_core" )
```

Comment:
These `blas_libs_list` match what's below for MKL. Can we just recognize Intel LLVM the same as the Intel compiler, and the test_mkl code would find it whether it's old MKL or oneMKL?

Reply:
I have merged these two searches.
0101bfa